sdks/java/core/src/test/java/org/apache/beam/sdk/io/AvroIOTest.java

```diff
@@ -627,30 +628,29 @@ public String getDefaultDestination() {
     public FilenamePolicy getFilenamePolicy(String destination) {
       return DefaultFilenamePolicy.fromStandardParameters(
           StaticValueProvider.of(
-              baseDir.resolve("file_" + destination + ".txt", StandardResolveOptions.RESOLVE_FILE)),
+              baseDir.resolve("file_" + destination + ".txt", RESOLVE_FILE)),
           null,
           null,
           false);
     }
   }
 
-  @Test
-  @Category(NeedsRunner.class)
-  public void testDynamicDestinations() throws Exception {
+  private void testDynamicDestinationsWithSharding(Integer numShards) throws Exception {
```

Comment:
<!--new_thread; commit:bd1ae5fc000332b3113096a9d6b58194924e60cc; resolved:0-->
nit: better to use an enum here instead of a magic value. Even if it hard codes the number of shards to 3.